apl/tabular-operators/externaldata-operator.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: externaldata
+description: 'This page explains how to use the externaldata operator in APL.'
+---
+
+The `externaldata` operator in APL allows you to retrieve data from external storage sources, such as Azure Blob Storage, AWS S3, or HTTP endpoints, and use it within queries. You can specify the schema of the external data and query it as if it were a native dataset. This operator is useful when you need to analyze data that is stored externally without importing it into Axiom.
+
+<Note>
+The `externaldata` operator currently supports external data sources with a file size of maximum 5 MB.
+
+In queries with `externaldata`, reference at least one Axiom dataset. Axiom doesn’t support APL queries that don’t reference Axiom datasets.
```

Comment:
this might change before release, will update if it does

Reply:
it did change! 
datasetless queries work now so this can be removed, 
example https://app.dev.axiomtestlabs.co/axiomers-ft83/query?qid=FkEPmd8Ufep-st9n59&relative=1